boards/arduino-mega2560/board.c

```diff
@@ -56,7 +56,7 @@ void board_init(void)
 void SystemInit(void)
 {
     /* initialize UART_0 for use as stdout */
```

Comment:
Me either.  Should we just open an issue for cleaning up the arduino boards.  The issue could be to rename the folder and simplify the architecture to potentially share more.  Or at make it so all arduinos have an stdio implementation